lib/pdn/pdn.c

```diff
@@ -121,24 +121,32 @@ static void on_cnec_esm(const char *notif)
 	}
 }
 
-static void on_cgev(const char *notif)
+static void parse_cgev(const char *notif)
```

Comment:
How about:
```suggestion
static void parse_cgev_pdn(const char *notif)
```
Contraposed to `parse_cgev_apn_rate_ctrl`?